src/main/java/sirius/biz/jobs/BasicJobFactory.java

```diff
@@ -296,4 +299,30 @@ public Map<String, String> buildAndVerifyContext(Function<String, Value> paramet
 
         return context;
     }
+
+    @Override
+    public JSON computeRequiredParameterUpdates(WebContext ctx) {
```

Comment:
unfortunately, no, because if I return a JSON String, the string will be quoted/escaped when written to the actual json response. I _could_ pass the StructuredOutput as parameter, BUT that would make it harder to include the json string in `job.html.pasta`